tools/deployment/package/docker-compose.yaml

```
@@ -125,3 +125,42 @@ services:
       "--concurrency", "${CLP_REDUCER_CONCURRENCY:-1}",
       "--upsert-interval", "${CLP_REDUCER_UPSERT_INTERVAL:-100}"
     ]
+
+  mcp-server:
+    <<: *service_defaults
+    hostname: "mcp_server"
+    profiles:
+      - "mcp"
```

since there's only one profile name assigned
```suggestion
    profiles: ["mcp"]
```